src/python-common/requirements.txt

```diff
@@ -4,3 +4,4 @@ mypy==0.782; python_version >= '3'
 pytest-mypy; python_version >= '3'
 pytest >= 2.1.3; python_version >= '3'
 pyyaml
+typing-extensions; python_version < '3.8'
```

Comment:
Will this fail sometime in the future, when we support Python versions equal to or higher than 3.8?

Reply:
No, typing extensions is only needed with python < 3.8.